### DISCUSSION TITLE
DYNAMITE Core Team Meeting 9th August

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#406] [Announcements]
Hello @dynamics-of-stellar-systems/dynamite-maintainers,

we are reconvening our DYNAMITE core team meeting on August 9th at the usual 1pm, either on zoom or in Glenn's office. We will discuss more the orbit colouring which is proceeding quickly it seems :) I can also try to walk you through the code that Ling sent me, if I manage to look at it in time haha
Please react if you are joining!

Cheers,
Alina

**Comment by prashjet** [comment]
Hi all. Apologies: Alina and I running a few minutes late
